bpf/init.sh

```diff
@@ -23,13 +23,11 @@ MTU=${13}
 # BPFFS_ROOT=${17}
 NODE_PORT=${18}
 # NODE_PORT_BIND=${19}
-MCPU=${20}
-NR_CPUS=${21}
+# MCPU=${20}
+# NR_CPUS=${21}
```

Comment:
Why can't we just remove all of these and their corresponding Go code?

Reply:
We've been doing it this way since we still need to backport some (but not all) init.sh changes like the socketlb downgrade code, and we don't want to cause unnecessary churn. For example, removing `LIB` would cause all of these to be re-numbered, which will cause pain during backports.